src/AppBundle/Listener/UserRegisterRedmine.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace AppBundle\Listener;
+
+use AppBundle\Entity\User;
+use Ekreative\RedmineLoginBundle\Security\RedmineUser;
+use Symfony\Component\Security\Core\SecurityContextInterface;
+use Symfony\Component\Security\Http\Event\InteractiveLoginEvent;
+use Doctrine\ORM\EntityManager;
+
+class UserRegisterRedmine
+{
+
+    private $entityManager;
+    private $security;
+
+    public function __construct(SecurityContextInterface $security, EntityManager $entityManager)
+    {
+        $this->security = $security;
+        $this->entityManager = $entityManager;
+    }
+
+    public function onInteractiveLogin(InteractiveLoginEvent $event)
+    {
+        /**
+         * @var RedmineUser $datauser;
+         */
+        $datauser = $this->security->getToken()->getUser();
+        $user = $this->entityManager->getRepository('AppBundle:User')->find($datauser->getId());
+        if(!$user) {
+            $user = new User();
```

Comment:
You wont need to do new User when you have set the param, you set user = $datauser

    parameters:
        ekreative_redmine_login.user.class: AppBundle\Entity\User